### ISSUE TITLE
Missing Type

### ISSUE THREAD

**Issue by dmtucker** [JasonSanford/geojsonlint.com#18]
There is no `type` property on the features in the [example](http://oldmapsonline.googlecode.com/svn-history/r76/trunk/mapanalyst/vectordemo/displacements.geojson) below, yet geojsonlint.com does not complain; however, Leaflet will not render it.

``` json
{
  "type": "FeatureCollection",
  "properties": {
    "title": "Displacements",
    "author": "MapAnalystOnline",
    "whatever": "date of creation? unique id generated from points?"
  },
  "features": [
    {
      "id": "1",
      "properties": {
        "displacement": 2.3,
        "color": "black"
      },
      "geometry": {
        "type": "GeometryCollection",
        "geometries": [
          {
            "type": "Point",
            "coordinates": [
              200,
              400
            ]
          },
          {
            "type": "LineString",
            "coordinates": [
              [
                200,
                400
              ],
              [
                230,
                440
              ]
            ]
          }
        ]
      }
    },
    {
      "id": "2",
      "properties": {
        "displacement": 3.7,
        "color": "red"
      },
      "geometry": {
        "type": "GeometryCollection",
        "geometries": [
          {
            "type": "Point",
            "coordinates": [
              1442,
              888
            ]
          },
          {
            "type": "LineString",
            "coordinates": [
              [
                1442,
                888
              ],
              [
                1450,
                825
              ]
            ]
          }
        ]
      }
    }
  ]
}
```
